release-notes/8.0/README.md

```diff
@@ -4,7 +4,7 @@
 
 - [Downloads](https://dotnet.microsoft.com/download/dotnet/8.0)
 - [Linux Package Managers](https://learn.microsoft.com/dotnet/core/install/linux)
-- [Containers](https://mcr.microsoft.com/catalog?search=dotnet/)
+- [Containers](https://mcr.microsoft.com/en-us/product/dotnet/sdk/about)
```

Comment:
Use culture-less links so that users will get redirected to the most appropriate content that's available for them.

```suggestion
- [Containers](https://mcr.microsoft.com/product/dotnet/sdk/about)
```